src/library_webgl.js

```diff
@@ -1461,7 +1461,7 @@ var LibraryGL = {
 #if MAX_WEBGL_VERSION >= 2
     if (GL.currentContext.version >= 2) { // WebGL 2 provides new garbage-free entry points to call to WebGL. Use those always when possible.
       if (GLctx.currentPixelUnpackBufferBinding) {
-        GLctx['compressedTexImage2D'](target, level, internalFormat, width, height, border, imageSize, data);
+        if (imageSize !== 0) GLctx['compressedTexImage2D'](target, level, internalFormat, width, height, border, imageSize, data);
```

Comment:
Let's remove the `!== 0` part in all of these, because Closure is unable to optimize these away. It is guaranteed that `imageSize` is always an integer, so `if (imageSize)` will be enough.